cfme/tests/migration_analytics/test_migration_analytics.py

```diff
@@ -0,0 +1,206 @@
+"""Tests around the Migration Analytics"""
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [
+    pytest.mark.ignore_stream("5.10"),
+    pytest.mark.tier(1),
+    test_requirements.migration_analytics
```

Comment:
I'd rather put these markers on the individual test cases and put `pytest.mark.manual` marker as a global marker.

Reply:
So here, ```pytest.mark.manual``` is added to individual test case because these test cases will be automated in this file only. This files is not for manual test case if you see it's name.
Also these below markers are common for all test cases in this file. Hence keeping global:
```
 pytest.mark.ignore_stream("5.10"),
    pytest.mark.tier(1),
    test_requirements.migration_analytics```

Reply:
Oh, I thought you were going to move test cases to a different file and then automate them. But if that is not the case, what you did is perfect.